test/Datadog.Trace.IntegrationTests/SendTracesToZipkinCollector.cs

```diff
@@ -12,12 +12,8 @@ public class SendTracesToZipkinCollector
 
         public SendTracesToZipkinCollector()
         {
-            var settings = new TracerSettings()
-            {
-                AgentUri = new System.Uri("http://localhost:9411/api/v2/spans")
-            };
-
-            var api = new ZipkinApi(settings);
+            var agentUri = new System.Uri("http://localhost:9411/api/v2/spans");
+            var api = new ZipkinApi(agentUri);
             var agentWriter = new AgentWriter(api, statsd: null);
             _tracer = new Tracer(settings, agentWriter, sampler: null, scopeManager: null, statsd: null);
```

Comment:
@pellared you still need the `settings` on this line.

Reply:
My bad... Totally missed it 🤦